convert/convert.go

```diff
@@ -229,7 +229,7 @@ func PairwiseFaToVcf(f []fasta.Fasta, chr string, out *fileio.EasyWriter, substi
 		} else if deletion {
 			pastStart = true
 			deletion = false
-			if !substitutionsOnly {
+			if !substitutionsOnly && deletionAlnPos >= 0 {
```

Comment:
Instead of ignoring this corner case, is it feasible to match the VCF specs here?

From the specs:

"Strings must include the base before the event (which must be reflected in the POS field), unless the event occurs at position 1 on the contig in which case it must include the base after the event"

Reply:
Do you have an idea of what that looks like for the alignment I showed above?
Would that work like this:
chr1 1 ATCA A

Reply:
Yes, that is my understanding.

Reply:
Added a TODO and updated the usage to reflect that we're not fixing this. This deletion in position 1 will break our parser I think

Reply:
Do we need to update the parser to treat mutations at position 1 as a special case?  Maybe not needed immediate, but it would be good to not lose site of this inability to handle deletions of the first base.  Can we discuss this at code review on Monday to try and understand how big of a fix it would be to handle these types of mutations?  I am ok to check in this improvement now, and have that discussion on Monday about a fix to the larger issue